data/locale/fr-FR.ini

```diff
@@ -30,8 +30,48 @@ NDIPlugin.Menu.OutputSettings="NDI™ - Sortie principale"
 NDIPlugin.OutputSettings.DialogTitle="Configuration des sorties NDI™"
 NDIPlugin.OutputSettings.GroupBox.Main="Sortie principale"
 NDIPlugin.OutputSettings.GroupBox.Preview="Sortie de prévisualisation"
+NDIPlugin.OutputSettings.CheckForUpdate="Vérifier les mises à jour"
+NDIPlugin.OutputSettings.TextCopied="Texte copié"
+NDIPlugin.OutputSettings.TextCopiedToClipboard="Texte copié dans le presse-papiers"
+
 NDIPlugin.FilterName="Sortie NDI™ dédiée"
 NDIPlugin.AudioFilterName="Sortie NDI™ dédiée (Audio seul)"
 NDIPlugin.PremultipliedAlphaFilterName="obs-ndi - Correction de simulation de transparence"
-NDIPlugin.LibError.Title="Bibliothèque NDI™ introuvable"
-NDIPlugin.LibError.Message="La bibliothèque NDI™ est introuvable sur votre système.<br><br>Téléchargez l'installateur ici:"
+
+NDIPlugin.Update.Title="Mise à jour de %1"
+NDIPlugin.Update.NewVersionAvailable="Nouvelle version %1 %2 disponible."
+NDIPlugin.Update.YourVersion="Votre version : <b>%1</b>"
+NDIPlugin.Update.ReleaseNotes="Notes de version :"
+NDIPlugin.Update.ReleaseNotes.None="Pas de notes de version disponibles."
+NDIPlugin.Update.ReleaseDate="Date de sortie :"
+NDIPlugin.Update.ContinueToCheckForUpdate="Continuer à vérifier les mises à jour lors des lancements futurs"
+NDIPlugin.Update.SkipThisVersion="Ignorer cette version"
+NDIPlugin.Update.RemindMeLater="Me le rappeler plus tard"
+NDIPlugin.Update.InstallUpdate="Installer la mise à jour"
+NDIPlugin.Update.CheckingForUpdate.Text="Vérification de la mise à jour de %1..."
+NDIPlugin.Update.CheckingForUpdate.Cancel="Annuler"
+NDIPlugin.Update.CheckingForUpdate.Error.Title="Erreur de vérification de mise à jour"
+NDIPlugin.Update.CheckingForUpdate.Error.Text="Erreur lors de la vérification de mise à jour :<br><pre>%1</pre>"
+NDIPlugin.Update.NoUpdateAvailable="Aucune mise à jour de %1 disponible"
+NDIPlugin.Update.YouAreUpToDate="Vous avez déjà la dernière version %1 de %2."
+
+NDIPlugin.Donate="Veuillez envisager de faire un don pour soutenir ce projet :"
```

Comment:
Suggested : 
NDIPlugin.Donate="Faites un don pour soutenir ce projet :"
More direct & to the point.

Reply:
No "Please"? :)
I did just notice that stray whitespace. :/